editor/src/components/canvas/dom-walker-caching.spec.browser2.tsx

```diff
@@ -101,11 +101,11 @@ describe('Dom-walker Caching', () => {
     expect(renderResult.getRecordedActions().map((a) => a.action)).toEqual([
       'SET_ELEMENTS_TO_RERENDER',
       'SET_CANVAS_FRAMES',
-      'UPDATE_FROM_WORKER',
+      'MERGE_WITH_PREV_UNDO',
```

Comment:
nitpick, but we should print and assert the wrapped actions here

Reply:
Yeah, that's fair, I'll fix it

Reply:
Ok I've used `simpleStringifyActions` here (after moving that function to a more sensible place and tweaking it a bit to give better indentation)